### PR TITLE
Start migration of Identity Providers settings screen

### PR DESCRIPTION
**Pull request by gaetanmaisse**
**Issue**

https://github.com/gravitee-io/issues/issues/6058

**Description**

 - Created a new Angular component for Identity Providers settings, this component display the list of IdP but doesn't have all the interaction yet
 - Created a new Angular service for OrganizationService and improve related types

**Screenshots**

image

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-shthorzqts.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6058-migrate-idp-list-screen/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-llkojvlsnu.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6058-migrate-idp-list-screen/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-axembsyfrq.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6058-migrate-idp-list-screen/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

📚 View the storybook of this branch [here](https://612657caa8e859003a8a6430-boszlnqyze.chromatic.com)

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6058-migrate-idp-list-screen/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: package.json [hunk 1]
```diff
@@ -158,6 +158,7 @@
     "build-storybook": "build-storybook",
     "serve": "webpack serve -c ./conf/webpack.conf.js --progress --hot",
     "serve:nightly": "BACKEND_ENV=nightly npm run serve",
+    "serve:element-zero": "BACKEND_ENV=element-zero-apim-api.cloud npm run serve",
```
Comment (ThibaudAV) [inline package.json:161]: 🤟

**Review by ThibaudAV — APPROVED** [review]

**Pull request merged**